build.gradle

```
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2020-2021 Exactpro (Exactpro Systems Limited)
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'com.palantir.docker' version '0.25.0' apply false
+    id 'org.jetbrains.kotlin.jvm' version "1.3.72"
+    id 'com.github.harbby.gradle.serviceloader' version '1.1.5'
+}
+
+apply plugin: 'kotlin'
+apply plugin: 'application'
+apply plugin: 'com.palantir.docker'
+
+ext {
+    sharedDir = file("${project.rootDir}/shared")
+    sailfishVersion = '3.2-SNAPSHOT'
+}
+
+group = 'com.exactpro.th2'
+version = release_version
+
+sourceCompatibility = 11
+targetCompatibility = 11
+
+ext {
+    junitVersion        = '5.4.2'
+    sailfishVersion     = '3.2-SNAPSHOT'
+    kotlin_version      = '1.3.72'
+    sharedDir           = file("${project.rootDir}/shared")
+}
+
+repositories {
+    mavenCentral()
+    mavenCentral()
+
+    maven {
+        name 'Bintray'
+        url 'https://dl.bintray.com/exactpro/th2'
+    }
+
+    maven {
+        name 'Sonatype_snapshots'
+        url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+    }
+
+    maven {
+        name 'Sonatype_releases'
+        url 'https://s01.oss.sonatype.org/content/repositories/releases/'
+    }
+
+    maven {
+        name 'Artifactory release'
+        url 'http://artifactory5.exp.exactpro.com/artifactory/libs-release'
+    }
+
+    maven {
+        name 'Artifactory snapshot'
+        url 'http://artifactory5.exp.exactpro.com/artifactory/libs-snapshot'
+    }
+
+    mavenLocal()
```

```suggestion

    maven {
        name 'Bintray'
        url 'https://dl.bintray.com/exactpro/th2'
    }

    maven {
        name 'Sonatype_snapshots'
        url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
    }

    maven {
        name 'Sonatype_releases'
        url 'https://s01.oss.sonatype.org/content/repositories/releases/'
    }
```
Please avoid using internal repository in opensource projects